playwright/test/helpers/contexts.ts

```diff
@@ -181,6 +181,21 @@ const workWithDigitalLocationAndLocationNote = async (
   await gotoWithoutCache(`${baseUrl}/works/a235xn8e`, page);
 };
 
+const workWithBornDigitalDownloads = async (
+  context: BrowserContext,
+  page: Page
+): Promise<void> => {
+  await context.addCookies(
+    requiredCookies.concat(
+      createCookie({
+        name: 'toggle_showBornDigital', // TODO: remove this when born digital work isn't behind a toggle anymore
```

Comment:
@LaurenFBaily I can't remember if it's something we want to do soon, but do we have a ticket for it where we can list the few TODOs that are similar to this ("Do X when BD goes live")?